strongbox-security/strongbox-authentication-registry/src/test/resources/etc/conf/eudst-strongbox-security-users.yaml

```diff
@@ -0,0 +1,13 @@
+users:
+  user:
+    - username: test-user
+      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
+      roles:
+        - ADMIN
+      securityTokenKey: user-secret
+    - username: test-deployer
+      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
+      roles:
+        - UI_MANAGER
+        - ARTIFACTS_MANAGER
+      securityTokenKey: deployer-secret
```

Comment:
Note: We need to create a follow-up task to be able to support multiple tokens.

Reply:
can you please clarify why multiple tokens needed?

Reply:
You may want to use the token from different services and for extra security, you could have different tokens for them. Multiple tokens are supported by various other applications (such as, for example, Sonarqube) and they're very convenient.

There's actually two follow-up tasks (with lower priority):
* Implement support for multiple tokens
* Implement token scopes (so that you could have tokens with read-only and write permissions) : For example, you want to be able to download artifacts from a remote system with this token, but drop write access for it.

We should look into this these in the future.